hyperbahn/endpoints/index.js

```diff
@@ -68,4 +70,12 @@ function setupEndpoints(clients, services) {
         //     handle(req, res, arg2, arg3);
         // }
     });
+
+    /*eslint no-unused-vars: 0*/
+    // thrift health endpoint
+    var tchannelAsThrift = new TChannelAsThrift({
+        source: '',
```

Comment:
Yes, that is something we have to live with for now ... until we have a real thrift IDL for Hyperbahn.